reflex-dom-core/src/Reflex/Dom/Prerender.hs

```diff
@@ -211,8 +212,10 @@ instance Monad m => MonadSample t (UnrunnableT js t m) where
   sample _ = unrunnable
 instance Monad m => MonadIO (UnrunnableT js t m) where
   liftIO _ = unrunnable
+#ifndef ghcjs_HOST_OS
 instance Monad m => MonadJSM (UnrunnableT js t m) where
```

Comment:
Oh does `MonadJSM` not exist then?

Reply:
it doesn't have `liftJSM'` in the ghcjs version

Reply:
http://hackage.haskell.org/package/jsaddle-0.9.6.0/docs/Language-Javascript-JSaddle-Monad.html#t:MonadJSM

MonadJSM == MonadIO on ghcjs

Reply:
Oh I think it would be better to also write `MonadIO UnrunnableT` then. Any platform-specificity of `UnrunnableT` is just a compile-error foot-gun lying in waiting.

Reply:
Ohh i get it now, the class itself goes away as it's just an alias.